docs/dev-setup.rst

```diff
@@ -90,7 +96,21 @@ To setup your local environment you should create a virtualenv and install the
 necessary requirements::
 
     mkvirtualenv --python=/usr/bin/python3.4 serviceinfo
```

Comment:
Sounds still not right, I think. :)

`which python3.4` gives me nothing—because I'm using pythonz to manage my Python versions.